endpoints/schemas/inventorySchemas.yaml

```diff
@@ -184,6 +184,12 @@ components:
                           type: string
                         rateCardName:
                           type: string
+						rateCardStartDate:
```

Comment:
the formatting is off a bit, I'm afraid it might impact the codegen

Reply:
Corrected the formatting. Please verify.